.stylelintrc.json

```diff
@@ -18,16 +18,57 @@
             }
 
         ],
+        "at-rule-empty-line-before": [
+            "always",
+            {
+                "except": [
+                    "first-nested",
+                    "blockless-after-blockless"
+                ],
+                "ignore": [
+                    "after-comment"
+                ]
+            }
+        ],
         "scss/at-import-no-partial-leading-underscore": null,
         "scss/at-mixin-argumentless-call-parentheses": "never",
         "order/order": [
             [
-                "declarations",
                 {
                     "type": "at-rule",
                     "name": "include",
                     "hasBlock": false
-                }
+                },
+                {
+                    "type": "at-rule",
+                    "name": "extend"
+                },
+                "custom-properties",
+                "declarations",
+                {
```

Comment:
presumably this would include @include hocus so should that go underneath mq?

Reply:
good point, I would say that should live with pseudo-states